app/config/overrides/config.json

```diff
@@ -1,14 +1,18 @@
 {
 	"debug": true,
 	"mapUnlockThreshold": 0,
-	"useLocalSessions": false,
+	"useLocalSessions": true,
 	"startRoom": "vestibule",
 	"welcomePage": {
 		"avatarSelectionEnabled": true,
 		"backgroundImagePath": "./js/images/bauhaus_darker.jpg",
 		"enterSpaceButtonText": "Party!",
 		"avatarSelectionText": "Pick your puck",
-		"headerHtml": "<br/><div class='bauhaus'><p>This is indeed a disturbing universe.</p><h1>Cabin Weekend is Dead. Long Live Cabin Fever.</h1>"
+		"headerHtml": "<br/><div class='bauhaus'><p>This is indeed a disturbing universe.</p><h1>Cabin Weekend is Dead. Long Live Cabin Fever.</h1>",
+		"auth": {
+			"iframe": "./js/iframes/TST_login.html",
```

Comment:
it's worth discussing i guess whether this is something we want to merge into master here or whether this should go into a fork

Reply:
I'm going to keep this here as a working example, but I have another branch for TST-specific customization that I'm guessing I should probably turn into a fork?